src/index.ts

```diff
@@ -56,6 +52,51 @@ export = (options: ManifestPluginOptions = {}): Plugin => ({
         entryPoints.set(input, output);
       }
 
+      for (const outputFilename in result.metafile.outputs) {
```

Comment:
I think all this can be simplified using the before/after suggestion mentioned previous in the pull request.

Reply:
It has been refactored and "condensed", It doesn't follow your style, let me know what you think, I could flatten the code and use a big for loop instead...

Reply:
I'm marking the comment as resloved since it's outdated :)